docs/pages/components/select.md

```diff
@@ -414,19 +414,17 @@ The disabled state can also be achieved by adding the `.is-disabled` class or th
 To make the `Select` component read-only, the readonly attribute needs to be added to the `fd-select__control` element. 
 This can also be done by adding the `.is-readonly` class or the `aria-readonly="true"` attribute.
 
-{% capture disabled-select %}
+{% capture readonly-select %}
 <div class="fd-popover">
-    <div class="fd-popover__control" aria-expanded="false" aria-haspopup="false" aria-readonly="true" readonly>
+    <div class="fd-popover__control" aria-expanded="false" aria-haspopup="false" aria-readonly="true">
         <div class="fd-select">
-            <div class="fd-select__control" role="button" aria-expanded="false" aria-haspopup="false" aria-readonly="true" readonly>
-                Select
-           </div>
+            <div class="fd-select__control" role="button" aria-expanded="false" aria-label="Select" aria-haspopup="false" aria-readonly="true"></div>
```

Comment:
I think this should have `tabindex="0"` since readonly inputs are focusable for screen-reader purposes. Also I believe you removed the selected value by mistake.
```
<div class="fd-select__control" role="button" aria-expanded="false" aria-label="Select" aria-haspopup="false" aria-readonly="true" tabindex="0">Potato</div>
```